portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,88 @@
+          through this website :) </p>
+
+      <h2> Projects & Contributions </h2>
+      <h3> Automatcher </h3>
+      <h5> May 2019 - August 2019 </h5>
+      <img src="/images/cplusplus.png" alt="c++ libraries logo" width="150" height="75">
+      <p><i> What is Automatcher?</i></p>
+      <p> As an intern on Google's C++ Libraries team, my podmate and I
+          worked on a tool called 
+          <a href = "https://docs.google.com/presentation/d/1aE6I6ZWN_BtyMYhmrPpYq-xEJz8Xrka_4Fj0mS_dqOk/edit#slide=id.p" target = "_blank">Automatcher</a>
+          . Automatcher is an AST matcher tool which
+          generates clang queries from user-provided code snippets. Usually complex queries
+          need to be written out by hand, but Automatcher automates this process. </p>
+      <i> What I did: </i>
+      <ul>
+      <li> Took ownership of the "hint" based approach to solving the problem of generating 
+          an Abstract Syntax Tree (AST) matcher from a user-friendly specification. </li>
+      <li>Implemented support for three different methods of annotating code fragments to guide generation of 
+          the AST matcher using C++. </li>
+      <li>Led the design and implementation of the first version of 
+          the essential data structure for representing "choice" in a code description.</li>
+      </ul>
+    <hr>
+      <h3> Submitty </h3>
+      <h5> September 2019 - December 2019 </h5>
+      <img src="/images/submitty.png" alt="submitty logo" width="150" height="75">
+      <p><i> What is Submitty? </i><p>
+      <p> <a href = "https://submitty.org/" target="_blank">Submitty</a> 
+          is an open source management, assignment submission, exam and
+           grading system from the Rensselaer Center for Open Source Software (RCOS), 
+           Department of Computer Science at Rensselaer Polytechnic Institute. </p>
+      <i> What I did: </i>
+      <p>Made open source contributions of fixing bugs and adding new features to the submission
+          server.</p>
+    <hr>
+      <h3> The Geo Effect </h3>
+      <h5> June 2017 - August 2017 </h5>
+      <img src="/images/gwc.png" alt="gwc logo" width="150" height="75">
+      <p><i> What is The Geo Effect? </i></p>
+      <p>The Geo Effect is an climate change awareness website I worked on as part of
+          the <a href="https://girlswhocode.com/" target = "_blank">Girls Who Code</a> Summer Immersion Program. </p>
+      <i>What I did:</i>
+      <p> Contributed to the backend of the website in implementing a blog for users to 
+          share their views on climate change.</p>
+    <hr>
+      <p> You can check out more of my work on 
+          <a href = "https://github.com/shreyabarua" target = "_blank">GitHub</a>.</p>
+
+      <h2> Cooking </h2>
+      <!--
+      <p>Click here to see some of my favorite foods:</p>
+      <button onclick="addRandomGreeting()">Favorite Foods</button>
+      <div id="greeting-container"></div> -->
```

Comment:
Is there a reason you're commenting this out for now? Just to save it for later? If so, I'd add a comment that says you'll implement this later.

Reply:
The Google standard for leaving a TODO comment looks like this:

`<!-- TODO: Implement gallery of favorite foods. -->`

or

`<!-- TODO(shreyabarua): Implement gallery of favorite foods. -->`

The second example just assigns ownership of the TODO to someone. In this case, yourself.

Reply:
I commented it out because I'm not sure I want this on my final portfolio but I haven't decided yet. I also just kept it for now in case I want to implement something similar to it in the future. I'll add a TODO to make it clearer.